### PR TITLE
Face Liveness Detection

### PR DESCRIPTION
**Pull request by ananyag309**
#68 the model i have added verify whether a person is physically present during the authentication process by analyzing facial features and eye blinks. 🔍

### Checklist

- [x] My code follows the current [project structure](https://github.com/yashasvini121/predictive-calc/blob/master/docs/project-structure.md)
- [x] I have thoroughly reviewed and updated the `requirements.txt` file to include any new packages
- [x] The `predict.py` file includes a properly implemented `model_details()` function, or the notebook contains this function to print a detailed model report. **The model will not be accepted without this function**, as it is essential for generating the necessary model details.
- [x] I have added relevant tests (if necessary).
- [x] I have added comments in the code where needed.
- [x] This PR is submitted under **Hacktoberfest**.
- [x] This PR is submitted under **GirlScript Summer of Code (GSSoC-Extd)**.

### CODE REVIEW

**Comment by Pranshu-jais** [comment]
@ananyag309  Please  add  Streamlit page   under pages/  directory

File: models/Face Liveness Detection/requirements.txt
Comment (yashasvini121) [inline models/Face Liveness Detection/requirements.txt:1]: Don't create a separate requirements.txt file, update the existing one.

File: models/Face Liveness Detection/main.py
Comment (yashasvini121) [inline models/Face Liveness Detection/main.py:1]: Follow the project structure.

File: models/Face Liveness Detection/dataset/haarcascade_frontalface_default.xml
Comment (yashasvini121) [inline models/Face Liveness Detection/dataset/haarcascade_frontalface_default.xml:1]: Save it in saved/models and change it to csv format

**Review by yashasvini121 — CHANGES_REQUESTED** [review]
Your work is incomplete, make the requested changes and create a page for your model.

File: models/Face Liveness Detection/requirements.txt
Comment (ananyag309) [reply models/Face Liveness Detection/requirements.txt:1]: dobe

**Comment by ananyag309** [comment]
@yashasvini121 can you please help me? i have converted xml file to csv and deleted requirement.txt what else need to be done

**Comment by yashasvini121** [comment]
@ananyag309, you can check the documentation to see if it helps. Additionally, in the `emotion-detection` branch, the [emotion detection model](https://github.com/yashasvini121/predictive-calc/tree/emotion-detector/models/emotion_detector) utilizes the camera feature, so you might find it useful to refer to that as well.

**Comment by ananyag309** [comment]
@yashasvini121 I have completed all the necessary requirements. If everything looks good, please proceed with merging them.

**Pull request closed without merging**